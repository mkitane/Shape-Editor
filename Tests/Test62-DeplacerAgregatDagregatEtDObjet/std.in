L ligne1 1 1 2 2
OA agregat1 ligne1
LIST
OA agregat2 agregat1 ligne1
LIST
MOVE agregat2 10 10
LIST
EXIT
