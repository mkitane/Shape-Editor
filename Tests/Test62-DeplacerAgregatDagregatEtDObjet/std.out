OK
OK
OA agregat1 ligne1 
L ligne1 1 1 2 2
OK
OA agregat1 ligne1 
OA agregat2 agregat1 ligne1 
L ligne1 1 1 2 2
OK
OA agregat1 ligne1 
OA agregat2 agregat1 ligne1 
L ligne1 11 11 12 12

